plugins/postgres/dbt/adapters/postgres/connections.py

```diff
@@ -23,6 +23,7 @@ class PostgresCredentials(Credentials):
     search_path: Optional[str] = None
     keepalives_idle: int = 0  # 0 means to use the default value
     sslmode: Optional[str] = None
+    application_name: Optional[str] = None
```

Comment:
What do you think about setting this to `'dbt'` by default? That's what we do on Snowflake and BigQuery today:

https://github.com/fishtown-analytics/dbt/blob/77c10713a325d2bee91d1822951ce5d91ccc3278/plugins/snowflake/dbt/adapters/snowflake/connections.py#L212

https://github.com/fishtown-analytics/dbt/blob/77c10713a325d2bee91d1822951ce5d91ccc3278/plugins/bigquery/dbt/adapters/bigquery/connections.py#L250

Reply:
> What do you think about setting this to `'dbt'` by default? That's what we do on Snowflake and BigQuery today:
> 
> https://github.com/fishtown-analytics/dbt/blob/77c10713a325d2bee91d1822951ce5d91ccc3278/plugins/snowflake/dbt/adapters/snowflake/connections.py#L212
> 
> https://github.com/fishtown-analytics/dbt/blob/77c10713a325d2bee91d1822951ce5d91ccc3278/plugins/bigquery/dbt/adapters/bigquery/connections.py#L250

Oh yeah, thats perfect. I can make the change.